### PR TITLE
orientdb 2.2.4

### PR DESCRIPTION
**Pull request by robfrank**
- [ x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [ ]x Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/orientdb.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Orientdb < Formula
   desc "Graph database"
   homepage "https://orientdb.com"
-  url "https://orientdb.com/download.php?file=orientdb-community-2.2.2.tar.gz"
```
Comment (DomT4) [inline Formula/orientdb.rb]: Why the move away from the upstream site here?

File: Formula/orientdb.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-  sha256 "1ae672fc15638526980e132b92fe1896867fa765da79eadf5adf7cdcad946f88"
+  # url "http://central.maven.org/maven2/com/orientechnologies/orientdb-community/2.2.4/orientdb-community-2.2.4.tar.gz"
```
Comment (DomT4) [inline Formula/orientdb.rb]: Commented out URL?

**Comment by DomT4** [comment]
Test is failing:
```
==> /usr/local/Cellar/orientdb/2.2.4/bin/orientdb start
Starting OrientDB server daemon...
==> curl -I localhost:2480
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed connect to localhost:2480; Connection refused
==> /usr/local/Cellar/orientdb/2.2.4/bin/orientdb stop
Stopping OrientDB server daemon...
Error: orientdb: failed
<0> expected but was
<7>.
```

**Comment by robfrank** [comment]
Hi, I'm the release manager of OrientDB, it's the first time I manage brew packaging, so apologise for my errors. 
I moved from the main site in favour of maven central because I used the same on Docker images AND the link will never change. I would like to be more consistent over different packaging/delivery systems. 
I'll clean the rb, stripping commented line.
For the error, I tested locally before doing the PRs, I'll check again.

**Comment by robfrank** [comment]
```
$ brew install --build-from-source orientdb
==> Downloading http://central.maven.org/maven2/com/orientechnologies/orientdb-community/2.2.4/orientdb-community-2.2.4.tar.gz
######################################################################## 100.0%
==> Downloading https://gist.githubusercontent.com/maggiolo00/84835e0b82a94fe9970a/raw/1ed577806db4411fd8b24cd90e516580218b2d53/orientdbsh
Already downloaded: /Users/frank/Library/Caches/Homebrew/orientdb--patch-d8b89ecda7cb78c940b3c3a702eee7b5e0f099338bb569b527c63efa55e6487e
==> Patching
==> Applying orientdbsh
patching file bin/console.sh
Hunk #1 succeeded at 18 with fuzz 1 (offset -3 lines).
patching file bin/orientdb.sh
Hunk #1 succeeded at 9 (offset 5 lines).
Hunk #2 succeeded at 25 (offset 5 lines).
Hunk #3 succeeded at 37 (offset 5 lines).
patching file bin/gremlin.sh
==> Caveats
Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`.
==> Summary
🍺  /usr/local/Cellar/orientdb/2.2.4: 226 files, 64.5M, built in 29 seconds
```

```
$ brew test orientdb
Testing orientdb
==> Using the sandbox
==> /usr/local/Cellar/orientdb/2.2.4/bin/orientdb start
==> curl -I localhost:2480
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   120    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
==> /usr/local/Cellar/orientdb/2.2.4/bin/orientdb stop
```

```
$ brew audit --strict orientdb
orientdb:
  * Stable: http://central.maven.org/maven2/com/orientechnologies/orientdb-community/2.2.4/orientdb-community-2.2.4.tar.gz should be `https://search.maven.org/remotecontent?filepath=com/orientechnologies/orientdb-community/2.2.4/orientdb-community-2.2.4.tar.gz`
Error: 1 problem in 1 formula

```

**Comment by DomT4** [comment]
You've managed to accidentally pull in a bunch of other commits here. You'll find rebasing a bit easier if you're using a unique branch rather than `master`. Would you find giving that a shot so the diff is only relevant to the actual changes being made?

**Comment by MikeMcQuaid** [comment]
Create a new pull request from a non-master branch and check it only has a single commit in it before creating. Thanks!

**Pull request closed without merging**